### PR TITLE
:book: Add initial APIs content

### PR DESCRIPTION
**Pull request by ncdc**

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Any reason not to merge?

/lgtm
/approve

**Comment by ncdc** [comment]
Would merge and iterate

**Comment by nrb** [comment]
Agreed on merging and iterating, this is better than what we have.

/lgtm

File: docs/content/en/concepts/apis-in-kcp.md [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+The identity's **hash** is similar to a public key; it is not private and there are times when other consumers of your
+exported APIs need to know and reference it.
+
+Given 2 Workspaces, each with its own `APIExport` that exports `widgets.example.kcp.dev`, kcp uses the identity hash (in
+a mostly transparent manner) to ensure the correct instances associated with the appropriate `APIResourceSchema` are
+served to clients. See [Run Your Controller](#Run-Your-Controller) for more information.
+
+#### Permission Claims
+
+An API provider may need to access APIs that come from other APIExports, or that are built in to kcp. This is done by
```
Comment (lionelvillard) [inline docs/content/en/concepts/apis-in-kcp.md:244]: this seems to contradict the documentation in the code:

```
// PermissionClaim identifies an object by GR and identity hash.
// Its purpose is to determine the added permissions that a service provider may
// request and that a consumer may accept and allow the service provider access to.
```

Comment (ncdc) [reply docs/content/en/concepts/apis-in-kcp.md:244]: What part looks contradictory?

Comment (lionelvillard) [reply docs/content/en/concepts/apis-in-kcp.md:244]: Confusing more than contradict. I would rephase it, maybe something like this: "API provider may need to access resources in consumer workspaces. These resources might come from other APIExports, ...."

Does that make sense?

Comment (ncdc) [reply docs/content/en/concepts/apis-in-kcp.md:244]: Yes, thanks for the clarification

Comment (ncdc) [reply docs/content/en/concepts/apis-in-kcp.md]: Pushed an update - WDYT?

Comment (lionelvillard) [reply docs/content/en/concepts/apis-in-kcp.md]: much better. Thanks!

**Comment by ncdc** [comment]
Re-tagging and un-holding
/unhold

**Pull request merged**